docs/reference/licensing/get-trial-status.asciidoc

```diff
@@ -16,16 +16,14 @@ This API enables you to check the status of your trial license.
 [float]
 ==== Description
 
-If you want to try the features that are included in a platinum license, you can 
-start a 30-day trial. 
+If you want to try all the subscription features, you can start a 30-day trial. 
```

Comment:
I'm not sure that `subscription features` is fully clear. Users may wonder which subscription it refers to, since we have multiple.

What about `existing features` instead?

Reply:
This is the recommended term in our style guide, but I'll double-check. @chandlergibb does this seem like the appropriate usage to you?

Reply:
@lcawl @bytebilly `subscription features` seems clear enough, if we're referring to Gold/Platinum/Enterprise collectively (Basic isn't a subscription).